tests/report.py

```diff
@@ -74,7 +74,8 @@ def get_version(base_url):
         for backend in ["pyciemss", "sciml"]:
             path = f"scenarios/{scenario}/{backend}"
             if os.path.exists(path):
-                scenario_spec[backend] = os.listdir(f"scenarios/{scenario}/{backend}")
+                scenario_spec[backend] = [f for f in os.listdir(f"scenarios/{scenario}/{backend}")
```

Comment:
This has been merged into main with your other PR so this branch can be rebased in a way where this change no longer shows up.